src/app/content/components/Toolbar/StudyGuidesButton.tsx

```diff
@@ -34,7 +35,9 @@ const StudyGuidesText = styled.span`
 // tslint:disable-next-line:variable-name
 const StudyGuidesButton = () => {
   const isEnabled = useSelector(studyGuidesEnabled);
-  if (!isEnabled) { return null; }
+  const hasStudyGuides = useSelector(studyGuidesSummary);
+
+  if (!isEnabled || !hasStudyGuides) { return null; }
```

Comment:
it wouldn't suck to have this extra check for if there is any data to happen inside the `studyGuidesEnabled` selector, but i don't have super strong feelings on this one. if we had more places checking to see if study guides were turned on i would care more